src/main/java/cz/it4i/parallel/ui/HPCSettingsScreenController.java

```diff
@@ -0,0 +1,225 @@
+
+package cz.it4i.parallel.ui;
+
+import java.io.File;
+
+import cz.it4i.parallel.runners.HPCSchedulerType;
+import cz.it4i.parallel.runners.HPCSettings;
+import cz.it4i.swing_javafx_ui.JavaFXRoutines;
+import javafx.beans.value.ObservableValue;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.SpinnerValueFactory;
+import javafx.scene.control.TextField;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import javafx.util.StringConverter;
+import lombok.Getter;
+import lombok.Setter;
+
+public class HPCSettingsScreenController extends AnchorPane {
+
+	@FXML
+	public TextField hostTextField;
```

Comment:
Consider make it private (as others)

Reply:
I made all the FXML properties in both "LocalSettingsScreenController" and "HPCSettingsScreenController" private.